src/libraries/Microsoft.PowerFx.Core/Public/Config/SymbolTable.cs

```diff
@@ -276,6 +278,11 @@ internal void AddFunction(TexlFunction function)
 
             // Add any associated enums 
             EnumStoreBuilder?.WithRequiredEnums(new TexlFunctionSet(function));
+        }
+
+        internal void AddUserFunction(TexlFunction function)
+        {
+            AddFunction(function);
         }
```

Comment:
Why do we need this? can't we  directly call AddFunction()?

Reply:
This was the original entry point. I've moved to another place and forgot to remove this.